config/jobs/cert-manager/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -1,49 +1,22 @@
+# THIS FILE HAS BEEN AUTOMATICALLY GENERATED
+# Don't manually edit it; instead edit the "cmrel" tool which generated it
+# Generated with: /home/ashley/workspace/cert-manager-release/bin/cmrel generate-tests --mode previous
```

Comment:
note: as for the previous periodics, this file has a messy diff because it removes 1.7.

If the changes in `release-next` and `current` look sane, this file is basically the same apart from the one extra `release-1.8` bazel test.